larray/core/axis.py

```diff
@@ -234,6 +236,60 @@ def by(self, length, step=None, template=None):
         """
         return self[:].by(length, step, template)
 
+    def astype(self, dtype: Union[str, np.dtype], casting: str = 'unsafe', inplace: bool = False) -> 'Axis':
+        """
+        Cast labels to a specified type.
+
+        Parameters
+        ----------
+        dtype: str or dtype
+            Typecode or data-type to which the labels are cast.
+
+        casting: str, optional
+            Controls what kind of data casting may occur. Defaults to ‘unsafe’.
+
+                * ‘no’ means the data types should not be cast at all.
```

Comment:
I don't really like those weird quotes. **IF** we start using them, we should use them everywhere. I am -0 on this but if you really want that, go ahead and make an issue or PR to do that change.